portfolio/src/main/webapp/script.js

```diff
@@ -41,6 +41,29 @@ async function htmlInject(templatePath, targetID) {
   }
 }
 
+/**
+ * Updates the comment section after a new comment is submitted
+ */
+async function updateCommentSection() {
+  const textElement = document.getElementById("comment-text");
+  const authorElement = document.getElementById("comment-author");
+
+  if (textElement && authorElement) {
```

Comment:
It still does, as this just checks whether the elements themselves are present on the webpage. Sometimes, for example, `document.getElementById("comment-text").value` gives an error because `.value` is called on `null` when the comment section is hidden or not on the webpage.

I've also removed the author feature in a more recent branch and replaced it with nicknaming and Authentication, so this is only temporarily including both fields anyways.